x-pack/platform/plugins/shared/cases/server/services/attachments/operations/get.ts

```diff
@@ -332,7 +332,7 @@ export class AttachmentGetter {
         acc.set(idBucket.key, {
           userComments: idBucket.reverse.comments.doc_count,
           alerts: idBucket.reverse.alerts.value,
-          events: idBucket.reverse.events.value,
+          events: idBucket.reverse.events.doc_count,
```

Comment:
The [event aggregation ](https://github.com/elastic/kibana/blob/4cafc183f532ef8e72a8b45c548f3079b7e2e647/x-pack/platform/plugins/shared/cases/server/services/attachments/operations/get.ts#L372-L376) doesn't work on bulk actions: with 4 events via bulk, and 1 event via row actions, the total is showing 2 instead of 5. 

the count is correct after i switch to cardinality

<img width="410" height="140" alt="image" src="https://github.com/user-attachments/assets/784bc5c2-191c-46dc-92ea-ec2624c179c8" />

<img width="422" height="435" alt="image" src="https://github.com/user-attachments/assets/30241ba0-24f0-42aa-bbe2-9f483699943b" />